### ISSUE TITLE
Option to persist the subjective billing queue

### ISSUE THREAD

**Issue by aaroncox** [AntelopeIO/leap#126]
After some conversations this evening, I'm opening this with the assumption that the subjective billing queue doesn't persist and needs to be re-established each time a node is started. If this is true, then there are situations where after a restart a node takes a significant amount of time for subjective billing to kick in for accounts with a high amount of allocated resources.

On WAX for example, around 2-4 hours after restarting our producer, it will spend time producing blocks with a low quantity of transactions while also subjectively billing and failing transactions. If it were possible to specify an option within the config to persist this information, the producer's nodeos process could be restarted and maintain that queue and not experience this recreation phase.

**Comment by aaroncox** [comment]
Some sample data, with the red box showing the drop in transactions per block on WAX after restarting.
![image](https://user-images.githubusercontent.com/677686/189276491-fec8251d-fe7d-4dcb-bddb-d6edd227588a.png)

**Comment by aaroncox** [comment]
Also to note during this period, this is what the [logs looked like](https://gist.githubusercontent.com/aaroncox/ef3c91ec33b3e855957c0f2d245c177f/raw/30d87fe4498a2050827386405b3fa1d3c002f1cb/sample.log).

**Comment by heifner** [comment]
You are correct, the subjective billing queue is not persisted. Subjective billing is therefore reset on stop/start.

**Comment by stephenpdeos** [comment]
We will revisit this ticket if it remains a problem after the release of Leap 3.2.0.

**Comment by spoonincode** [comment]
Closing this in favor of newer issue at AntelopeIO/spring#1610
